lib/apiservers/engine/backends/portlayer/container_proxy.go

```diff
@@ -68,17 +70,21 @@ import (
 	"github.com/vmware/vic/pkg/vsphere/sys"
 )
 
+// VicContainerProxy interface
 type VicContainerProxy interface {
 	CreateContainerHandle(imageID string, config types.ContainerCreateConfig) (string, string, error)
```

Comment:
For my own sanity - is this Get to get an existing container or Create to create a new container?